decred/decred/crypto/secp256k1/curve.py

```diff
@@ -210,10 +210,10 @@ def splitK(self, k):
         # being a bottleneck.
         # c1 = round(b2 * k / n) from step 4.
         # Rounding isn't really necessary and costs too much, hence skipped
-        c1 = (self.b2 + k) // self.N
+        c1 = (self.b2 * k) // self.N
```

Comment:
This (and the similar one below) is a bugfix: the original Go code has a bigInt.Mul here, and tests fail without this change.

Reply:
Weird that we never hit an issue with this with signature verification. Hmmm.